### DISCUSSION TITLE
Update on the development branch

### DISCUSSION THREAD

**Discussion by kaiyux** [NVIDIA/TensorRT-LLM#1056] [Announcements]
Hi,

The TensorRT-LLM team is pleased to announce that we are pushing an update to the [development](https://github.com/NVIDIA/TensorRT-LLM/tree/main) branch (and the [Triton backend](https://github.com/triton-inference-server/tensorrtllm_backend/tree/main)) this February 6th, 2024.

This [update](https://github.com/NVIDIA/TensorRT-LLM/pull/1055) includes:

* Model Support
  - Skywork model support
* Features
  - More head sizes support for LLaMA-like models
    - Ampere (sm80, sm86), Ada (sm89), Hopper(sm90) all support head sizes [32, 40, 64, 80, 96, 104, 128, 160, 256] now.
* Bug fixes
  - Minor bug fixes
* Documentation
  - Add blog: Speed up inference with SOTA quantization techniques in TRT-LLM
  - A bunch of documentation fixes for the new building workflow

Thanks,
The TensorRT-LLM Engineering Team

**Comment by cody-moveworks** [comment]
When is the next official release coming out? Very eager to use it :)

**Reply by juney-nvidia** [reply]
Hi @cody-moveworks ,

The team is working hard with the next official release, the current release plan targets the end of February, but you know for software release it is not surprising to have the surprise:)

BTW, you can always pick up our latest main branch to try with the new features.

Thanks
June
